OracleUnifiedDirectory/dockerfiles/buildDockerImage.sh

```diff
@@ -0,0 +1,184 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
+#
+# Licensed under the Universal Permissive License v 1.0 as shown at 
+# https://oss.oracle.com/licenses/upl.
+#
+# Description: script to build a Docker image for Oracle Unified Directory
+#
+
+usage() {
+cat << EOF
+
+Usage: buildDockerImage.sh -v [version] -d [domain-type] [-s]
+Builds a Docker Image for Oracle Unified Directory
+
+Parameters:
+   -h: view usage
+   -v: Release version to build. Required. E.g 12.2.1.4.0
+   -s: skips the MD5 check of packages (DEFAULT)
+
+Copyright (c) 2020: Oracle and/or its affiliates. All rights reserved.
+Licensed under the Universal Permissive License v 1.0 as shown at 
+https://oss.oracle.com/licenses/upl.
+EOF
+exit 0
+}
+
+
+#=============================================================
+checkFilePackages() {
+  echo "INFO: Checking if required packages are present..."
+
+  jarList=`grep -v -e "^#.*" oud.download | awk '{print $2}'`
+  for jar in ${jarList}; do
+     if [ -s ${jar} ]; then
+       echo "INFO:   ${jar} found. Proceeding..."
+     else
+       cat > /dev/stderr <<EOF
+
+ERROR: Install Distribution ${jar} not found in
+  `pwd`
+  The following are required to proceed.
+EOF
+       cat oud.download
+       exit 1
+     fi
+  done
+}
+
+#=============================================================
+checksumPackages() {
+  if [ "${SKIPMD5}" -eq 1 ]; then
+    echo "INFO: Skipped MD5 checksum as requested"
+    return
+  fi
+
+  echo "INFO: Checking if required packages are valid..."
+
+  md5sum -c oud.download 2> /dev/null
+  if [ "$?" -ne 0 ]; then
+    cat <<EOF
+
+ERROR: MD5 for required packages to build the ${VERSION}
+       image did not match. Please make sure to download
+       or check the files in the ${VERSION} folder.
+EOF
+    cat oud.download
+    echo " "
+    exit $?
+  fi
+}
+
+#Parameters
+VERSION="NONE"
+SKIPMD5=0
```

Comment:
This implies that the default is NOT to skip the MD5 check so your usage doc is wrong.

Reply:
updated usage doc